build-system/tasks/visual-diff/index.js

```diff
@@ -826,7 +826,7 @@ async function ensureOrBuildAmpRuntimeInTestMode_() {
   }
 
   if (argv.nobuild) {
-    const isInTestMode = /AMP_CONFIG=\{(?:.+,)?"test":true\b/.test(
+    const isInTestMode = /AMP_CONFIG=\{(?:.+,)?"test":(true|!0)\b/.test(
```

Comment:
Is this due to `esbuild`?

Reply:
Thank you for asking! In a previous iteration of this PR I had been minifiying _after_ applying the amp config. I no longer do that though and can therefore remove this